blog/_posts/2018-10-05-conference-reactjsday-verona.md

```diff
@@ -0,0 +1,148 @@
+
+**First time in Italy**
+
+The organisers are italian and they very much want to put conferences on that are financially accessible to all their community and to get local italian companies involved.
+
+So to travel to a country not knowing italian added more apprenhension. I should not have been worried, all the talks were in English and were very clear. The italians are great hosts. I had a lovely time visiting Verona, the public transport is easy to understand, the local taxis are plentyful and the hotel where I was staying and hosting the conference were marvellous. They even found a hdmi cable so I could watch my netflix in comfort!
+
+**Feeling nervous before the conference**
+
+My negative inner voice starts chattering away - it maybe too hard, will I understand what they are talking about? Is it going to be at such an advanced level? What are the people going to be like? I should quit before they find out I am a fraud. I can have some or all of these negative thoughts before I go.
+
+So how do I gain control of my mindset and be ready to learn and have fun.
+Affirmations, practice, support.
+
+**Affirmations:** I am not meant to know it all. It is good for all to have different perspectives and diversity. If you are learning you are going to be challenged. It is ok to say you don't know something, in fact it helps the speakers go over details. Keep going to conferences. Not all are the same. They have different communities, different speakers, but the same goal that you have a great time, learn and meet the community.
+
+**Practice:** Keep going to conferences you will get more comfortable.
+
+**Support:** Get to know someone who will support you either remotely like Ana Sofia, or reach out to someone who is going too.
+
+### Summary
+
+I have attended a few conferences now. This one like the others is seriously lacking in equal representation. Therefore it is even more important as a woman to go, and that RGSoC continues to do what it does. Women need to start feeling comfortable being the minority until things change, standing out but also having a valid reason to attend. Thank you to all the conferences that are proactively addressing this balance.
+
+There were great live demos, live recordings, humour, reassurance and support to each other and a great community.
+
+Conferences like this brige the gap between reality and vision, best practice and deadlines, passion and ideas of where we are heading in the future.
+
+This was a conference about the framework that our project for RGSoC, Bahmni used. It would be exciting to find out more about this. It confirmed what we found difficult and similar pain points. I found out alternative ways of solving problems we had on our project. It also confirmed to me that I had grown as a developer and understood more than what I thought!
+
+### Speakers and Schedule
+
+There were so many great speakers, thank you to the organisers @grusp and @\_fevr
```

Comment:
I assume that 'grusp' and 'fevr' are twitter handles of conf organizers. Can you please add links to their twitter handles if you want to tag them. Markdown doesn't really transform it into links automatically. Thanks.